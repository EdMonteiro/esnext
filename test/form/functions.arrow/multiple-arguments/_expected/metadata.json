{
  "functions.arrow": {
    "functions": [
      {
        "type": "FunctionExpression",
        "id": null,
        "params": [
          {
            "type": "Identifier",
            "name": "n",
            "typeAnnotation": null
          },
          {
            "type": "Identifier",
            "name": "i",
            "typeAnnotation": null
          }
        ],
        "body": {
          "type": "BlockStatement",
          "directives": [],
          "body": [
            {
              "type": "ReturnStatement",
              "argument": {
                "type": "BinaryExpression",
                "operator": "*",
                "left": {
                  "type": "Identifier",
                  "name": "n",
                  "typeAnnotation": null
                },
                "right": {
                  "type": "Identifier",
                  "name": "i",
                  "typeAnnotation": null
                }
              }
            }
          ]
        },
        "generator": false,
        "async": false,
        "returnType": null,
        "typeParameters": null
      }
    ]
  }
}