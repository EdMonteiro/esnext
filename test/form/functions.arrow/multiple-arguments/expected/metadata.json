{
  "functions.arrow": {
    "functions": [
      {
        "type": "FunctionExpression",
        "id": null,
        "generator": false,
        "expression": false,
        "params": [
          {
            "type": "Identifier",
            "name": "n"
          },
          {
            "type": "Identifier",
            "name": "i"
          }
        ],
        "body": {
          "type": "BlockStatement",
          "body": [
            {
              "type": "ReturnStatement",
              "argument": {
                "type": "BinaryExpression",
                "operator": "*",
                "left": {
                  "type": "Identifier",
                  "name": "n"
                },
                "right": {
                  "type": "Identifier",
                  "name": "i"
                }
              }
            }
          ]
        }
      }
    ]
  }
}